templates/code-page.html

```diff
@@ -23,8 +32,31 @@ <h2>{{ commands_title }}</h2>
   </div>
 </div>
 <div tabindex="0" class="flex-grow p-8">
-  {# When we're NOT in adventure mode but DO have adventure-assignments, render them in tabs #}
-  {% include "incl-adventure-tabs.html" %}
+  {% if adventure_name %}
```

Comment:
We should use the version in `test` here instead of this one.

Reply:
Fixed!